doc/articles/get-started-vscode-devkit.md

```diff
@@ -0,0 +1,52 @@
+---
+uid: Uno.GetStarted.vscode.DevKit
+---
+
+## How to switch to C# DevKit Mode
```

Comment:
This should be an h1, I think. And next headings should also be reduced by one level

Reply:
yes, that would be the correct markdown
the structure was copied from existing documents and, since it's processed, I'm not sure of end-result. Being identical to other documents was safer.